bpf/lib/nodeport.h

```diff
@@ -1905,8 +1913,17 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
  *
  * CILIUM_CALL_IPV{4,6}_NODEPORT_REVNAT is plugged into CILIUM_MAP_CALLS
  * of the bpf_host, bpf_overlay and of the bpf_lxc.
+ * arguments
+ * ctx                     : pointer to packet context buffer.
+ * ifindex                 : for Nodeport reply traffic packet, this function writes
+ *                           interface index value at this ifindex pointer.
+ * is_revnat_at_egress_host: boolean flag to indicate perform reverse NAT only for
+ *                           egress nodeport traffic at host interface.
+ * return                  : On success, returns CTX_ACT_OK code to pass packet to next stack.
+ *                           on error, returns appropriate error code.
```

Comment:
nit: I'd probably drop those bits given we don't have such comments elsewhere (unless we plan to add them)

Reply:
sure.